### PR TITLE
Optimize java doc in ShardingRule class

### PR DESCRIPTION
**Pull request by KSMASTERS**
Fixes #10461   .

Changes proposed in this pull request:
- Optimize java doc in ShardingRule class
-
-

### CODE REVIEW

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java [hunk 1]
```diff
@@ -326,10 +326,10 @@ public boolean isSingleTablesInSameDataSource(final Collection<String> logicTabl
     }
     
     /**
-     * Judge if there is at least one table rule for logic tables.
+     * Judge if there is at least one table rule for logic tables or not.
```
Comment (strongduanmu) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java:329]: @KSMASTERS `Judge whether a table rule exists for logic tables or not` may be better.

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java [hunk 2]
```diff
@@ -339,18 +339,18 @@ public boolean tableRuleExists(final Collection<String> logicTableNames) {
...
-     * @return whether single table rule exists for logic tables
+     * @return whether single table rule exists for logic tables or not
      */
     public boolean singleTableRuleExists(final Collection<String> logicTableNames) {
         return singleTableRules.keySet().stream().anyMatch(logicTableNames::contains);
     }
     
     /**
-     * Judge is sharding column or not.
+     * Judge if a column is sharding column or not.
```
[the remark this answers is not in the document]
Comment (KSMASTERS) [reply shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java:349]: It seems that I have been unable to append the commit again on the original basis, I modified the local code and did not add another branch to do other work, so the append commit again did not record

[the remark this answers is not in the document]
Comment (KSMASTERS) [reply shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java:349]: > @KSMASTERS `Judge whether a column is sharding column or not.` may be better.

It seems that I have been unable to append the commit again on the original basis, I modified the local code and did not add another branch to do other work, so the append commit again did not record

[the remark this answers is not in the document]
Comment (strongduanmu) [reply shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java:349]: > > @KSMASTERS `Judge whether a column is sharding column or not.` may be better.
> 
> It seems that I have been unable to append the commit again on the original basis, I modified the local code and did not add another branch to do other work, so the append commit again did not record

@KSMASTERS You only need to modify the question, then submit your change files and push it to your repository.

**Comment by tristaZero** [comment]
Hi @KSMASTERS Could you have a look?

**Comment by tristaZero** [comment]
@KSMASTERS Hi, This PR needs your attention, could you have a look?

**Comment by tristaZero** [comment]
Since there is no any feedback from @KSMASTERS I have to close this one.

**Pull request closed without merging**